nilearn/plotting/html_document.py

```diff
@@ -44,6 +46,7 @@ def __init__(self, html, width=600, height=400):
         self.height = height
         self._temp_file = None
         self._check_n_open()
+        self._proc = None
```

Comment:
maybe we could use a more explicit name such as `self._temp_file_removing_proc`?